tests/ClientBuilderTest.php

```diff
@@ -56,7 +56,7 @@ public function testSetUriFactory(): void
         /** @var UriFactory|MockObject $uriFactory */
         $uriFactory = $this->createMock(UriFactory::class);
 
-        $clientBuilder = new ClientBuilder(['dsn' => 'http://public:secret@example.com/sentry/1']);
+        $clientBuilder = new ClientBuilder(new Options(['dsn' => 'http://public:secret@example.com/sentry/1']));
```

Comment:
To simplify the tests I would use the `create` method so that we avoid creating the `Options` object everywhere